src/foam/u2/view/RichChoiceView.js

```diff
@@ -720,7 +720,7 @@ foam.CLASS({
           });
 
           var summary = this.fullObject$.map(o => {
-            return o ? o.toSummary() : this.defaultSelectionPrompt;
+            return ( o && o.toSummary() ) ? o.toSummary() : this.defaultSelectionPrompt;
```

Comment:
Better to not compute toSummary twice. So:

return (o && o.toSummary()) || this.defaultSelection;